Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -3061,6 +3104,74 @@ def create_site_command(client: Client, name: str, description: str | None = Non
     )
 
 
+def get_site_included_targets(client: Client, site_id: str | None = None, site_name: str | None = None):
+    """
+    Get site included targets.
+
+    Args:
+        client (Client): Client to use for API requests.
+        site_id (str | None, optional): ID of the site to get the included targets for.
+        site_name (str | None, optional): Name of the site to get the included targets for.
+            Can be used instead of "site_id".
+    """
+    site = Site(
+        site_id=site_id,
+        site_name=site_name,
+        client=client,
+    )
+
+    response_data = client.get_site_included_targets(
+        site_id=site.id
+    )
+    included_targets = response_data['addresses'] if 'addresses' in response_data else []
+
+    return CommandResults(
+        readable_output=f"Included targets for site ID: {site.id} ->  {response_data}",
+        outputs_prefix="Nexpose.SiteIncludedTargets",
+        outputs_key_field="included_targets",
+        outputs={"included_targets": included_targets},
+        raw_response=response_data,
+    )
+
+
+def update_site_included_targets(client: Client, site_id: str | None = None, site_name: str | None = None,
+                                 targets: str | None = None):
+    """
+    Update site included targets.
+
+    Args:
+        client (Client): Client to use for API requests.
+        site_id (str | None, optional): ID of the site to update the included targets for.
+        site_name (str | None, optional): Name of the site to update the included targets for.
+            Can be used instead of "site_id".
+        targets (str | None): Comma separated list of targets to be updated.
+    """
+    site = Site(
+        site_id=site_id,
+        site_name=site_name,
+        client=client,
+    )
+
+    targets = targets.split(',')
+
+    client.update_site_included_targets(
+        site_id=site.id, targets=targets
+    )
+
+    response_data = client.get_site_included_targets(
+        site_id=site.id
```

Comment:
The `get_site_included_targets` expect site_id of type int as you wrote, but you provide str. Please fix.